### PR TITLE
TMQ8 - set up navigation

### PR DESCRIPTION
**Pull request by broy94**
https://byop-team.workast.io/task/bc09ebec944dc54d939f425446fd6c11

Setting up navigation with initial bootstrap layout (to be adjusted).
Deleted 'About' component.
Renamed 'Open Hours' component.

### CODE REVIEW

File: src/app/about/about.component.spec.ts [hunk 1]
```diff
@@ -1,25 +0,0 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (ptomo) [inline src/app/about/about.component.spec.ts]: Don't delete these please, it provides a skeleton for the testing part. You should delete the it(...) methods if you don't like them.

File: src/app/about/about.component.ts [hunk 2]
```diff
@@ -1,15 +0,0 @@
-import { Component, OnInit } from '@angular/core';
```
Comment (ptomo) [inline src/app/about/about.component.ts]: Don't remove the content of the .ts files pleeeeeeeease

**Comment by broy94** [comment]
https://byop-team.workast.io/task/b6cd6aa7636e7baea0fc2ef3a42762c5

File: package-lock.json [hunk 3]
```diff
@@ -1364,9 +1364,14 @@
       "integrity": "sha1-OciRjO/1eZ+D+UkqhI9iWt0Mdm8=",
       "dev": true,
       "requires": {
-        "hoek": "5.0.3"
+        "hoek": "2.16.3"
```
Comment (ptomo) [inline package-lock.json]: change back the version number, github will say it's unsecure

**Pull request merged**